### PR TITLE
issue: provide hashtags field

### PR DESCRIPTION
**Pull request by sterchelen**
- We want to share, let's use hashtags as a way to improve it

### CODE REVIEW

**Comment by sterchelen** [comment]
@pabluk didn't see where you push to twitter... Maybe we could add on the gh workflow a test on how long twitter_body is.

**Comment by pabluk** [comment]
Thanks for sharing this idea @sterchelen!

For a general overview of this change, in this PR or maybe another we also need to change the script [bin/create-content.py](https://github.com/sre-paris/reliability.re/blob/main/bin/create-content.py) that creates the Hugo content and were the tweet length validation could happen.

To make available to bots and parsers the new content, it needs to be included into the RSS/Atom feed so its [template](https://github.com/sre-paris/reliability.re/blob/main/layouts/_default/rss.xml) and [extension](https://github.com/sre-paris/reliability.re/blob/main/content/rss-ns-spec-1.0.md) need to be updated as well.

> @pabluk didn't see where you push to twitter... Maybe we could add on the gh workflow a test on how long twitter_body is.

Yep I need to push here or into other repo, the code of serverless functions used for Twitter and Telegram, they were created on the online editor of Google Cloud Functions. But that's the easy part that function just parse the RSS feed to find a field and post it using the Twitter API.

File: .github/ISSUE_TEMPLATE/new-content.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ about: Submit content to be reviewed and published
 title: Content's title
 labels: content
 assignees: ''
-
+twitter_body: Specific body for twitter
+twitter_hashtags: ['list', 'of', 'hashtags']
```
Comment (pabluk) [inline .github/ISSUE_TEMPLATE/new-content.md:8]: In which way do propose to use this list of hashtags? They will be append to the of tweet? or do you see other use case?

Comment (sterchelen) [reply .github/ISSUE_TEMPLATE/new-content.md:8]: They will be happened to the tweet... with defaults ones like "#sre" ...

**Comment by pabluk** [comment]
If it helps, I've just included a diagram from the last meetup with the workflow here https://github.com/sre-paris/reliability.re#publishing-workflow

**Comment by sterchelen** [comment]
> To make available to bots and parsers the new content, it needs to be included into the RSS/Atom feed so its [template](https://github.com/sre-paris/reliability.re/blob/main/layouts/_default/rss.xml) and [extension](https://github.com/sre-paris/reliability.re/blob/main/content/rss-ns-spec-1.0.md) need to be updated as well.

Why would we need to include it into the rss/atom feed?

**Comment by sterchelen** [comment]
@pabluk I removed twitter specifics... 
IMO we should do the shortened string into the google function by using the original content body.
On the other hand, I have let the `hashtags` field... Could be used for twitter... facebook... linkedin.

File: bin/create-content.py [hunk 2]
```diff
@@ -63,6 +63,9 @@ def main(filename):
     if twitter_username:
         content += "twitter_username: %s\n" % twitter_username
 
+    hashtags = data["issue"]["hashtags"]
```
Comment (pabluk) [inline bin/create-content.py:66]: Interesting I didn't know that Github injects markdown frontmatter fields into the event payload :+1:

**Review by pabluk — APPROVED** [review]

**Comment by pabluk** [comment]
Let's try like this and I'll add the Google Functions in https://github.com/sre-paris/reliability.re-functions/ asap to share the way they work and allow everyone to improve them.

**Comment by pabluk** [comment]
Thanks @sterchelen for your contribution!

**Pull request merged**